src/index.html

```diff
@@ -237,22 +237,30 @@ <h2>SKILLS</h2>
       <div>
         <section>
           <h2>QUICK ID</h2>
-          <ul>
-            <li><b>Phone</b><a href="tel:123-456-7890">123-456-7890</a></li>
-            <li>
-              <b>Email</b><a href="mailto:info@mysite.com">info@mysite.com</a>
-            </li>
-            <li>
-              <b>Website</b
-              ><a href="http://www.wix.com" target="_blank">www.wix.com</a>
-            </li>
-            <li>
-              <b>Address</b>
-              <address>
-                500 Terry Francois Street San Francisco, CA 94158
-              </address>
-            </li>
-          </ul>
+          <table id="table1">
+            <tr>
+              <th>Phone</th>
+              <a href="tel:123-456-7890"><td>123-456-7890</td></a>
```

Comment:
`a` tag can't be place outside the `td` tag. Correct way is
`<td><a href="tel:123-456-7890">123-456-7890</a></td>`

Please fix all the similar issues below.

Reply:
Thank you, I will fix it.